openfecli/fetchables.py

```diff
@@ -5,6 +5,18 @@
 _EXAMPLE_NB_BASE = ("https://raw.githubusercontent.com/"
                     "OpenFreeEnergy/ExampleNotebooks/main/")
 
+RBFE_SHOWCASE = URLFetcher(
+    resources=[
+        (_EXAMPLE_NB_BASE + "openmm_rbfe", "inputs/ligands.sdf"),
+        (_EXAMPLE_NB_BASE + "openmm_rbfe", "inputs/Jnk1_ligands.sdf"),
+        (_EXAMPLE_NB_BASE + "openmm_rbfe", "inputs/181L_mod_capped_protonated.pdb"),
+    ],
+    short_name="rbfe-showcase",
+    short_help="Inputes needed for the RBFE Showcase Notebook",
```

Comment:
```suggestion
    short_help="Inputs needed for the RBFE Showcase Notebook",
```

But really, is there any reason not to also include the notebook itself? I suppose technically this will overwrite something on colab, but should overwrite with the same thing, no? Including the notebook makes this useful for regular users, too.